app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,24 +58,26 @@ module SupportsFeatureMixin
   extend ActiveSupport::Concern
 
   QUERYABLE_FEATURES = {
-    :associate_floating_ip    => 'Associate a Floating IP',
-    :control                  => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
-    :cloud_tenant_mapping     => 'CloudTenant mapping',
-    :cinder_service           => 'Cinder storage service',
-    :delete                   => 'Deletion',
-    :disassociate_floating_ip => 'Disassociate a Floating IP',
-    :discovery                => 'Discovery of Managers for a Provider',
-    :evacuate                 => 'Evacuation',
-    :events                   => 'Query for events',
-    :launch_cockpit           => 'Launch Cockpit UI',
-    :live_migrate             => 'Live Migration',
-    :migrate                  => 'Migration',
-    :provisioning             => 'Provisioning',
-    :reconfigure              => 'Reconfiguration',
-    :regions                  => 'Regions of a Provider',
-    :resize                   => 'Resizing',
-    :retire                   => 'Retirement',
-    :smartstate_analysis      => 'Smartstate Analaysis',
+    :associate_floating_ip        => 'Associate a Floating IP',
+    :control                      => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
+    :cloud_tenant_mapping         => 'CloudTenant mapping',
+    :cloud_volume_backup_create   => 'CloudVolume backup creation',
+    :cloud_volume_backup_restore  => 'CloudVolume backup restore',
+    :cinder_service               => 'Cinder storage service',
```

Comment:
Yeah, I'm not a fan of this here ... (or the `:swift_service` below).

Though, I think it's less about having a global registry and more about having a way for providers to expose their features into the registry.

We should think about how to allow providers to expose their features, so that they're still queryable and reportable and also allow cross manager dependencies like we're seeing with the [`StorageManager::CinderManager`](https://github.com/ManageIQ/manageiq/blob/77d6be26f4d0870e26d1a5ded2a6340cf5be5493/app/models/manageiq/providers/storage_manager/cinder_manager.rb#L36).

In that sense, we would only have core MIQ features in the main registry, and allow providers to register features.